add: basic ridable element and components & fireman carry

NightDawnFox opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Перенос элементов и компонентов езды на наш билд. При этом они не используются, весь транспорт работает по старым правилам (мне было лень делать это сейчас)
Добавление хвата пожарного. Позволяет таскать людей. Всё прямиком с ТГ!
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
Пока ничего нет, чуть позже может быть
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений

https://github.com/user-attachments/assets/2675f6f0-3d2a-4f90-ae3f-4686cbcca74f

![tenor](https://github.com/user-attachments/assets/3bfdebce-1bca-4bfd-bf68-cd3420d0ce69)

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

NightDawnFox committed:
v0.5

NightDawnFox commented:
открыто для ревью, не для мерджа

Gottfrei commented:
сперва сделай транспорт на компоненте, отдельно, а затем уже приступай к остальному.

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
багфиксы пожарного хвата

NightDawnFox committed:
v0.6

TODO:
Перевести остальной транспорт под компонент, а именно:
-ambulance
-janicart
-motorcycle
-secway
-snowmobile
-speedbike
-sportscar
- lava & dragonboats

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
ambulance done

NightDawnFox committed:
janicart

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
еще 4 строчки кода, юхуу!

NightDawnFox committed:
готово

NightDawnFox commented:
Весь текущий транспорт в игре переделан под компонент, включая щитспавновый. Добавлять что-то новое пока глупо

ROdenFL commented:
Добавь клоун кар

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
совместимость с новой

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

Daeberdir commented:
![image](https://github.com/user-attachments/assets/2f6beb93-bc76-4891-b30d-c653bbc526f6)

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

NightDawnFox committed:
Merge remote-tracking branch 'upstream/master220' into riding

Rafnik02 commented:
зюзя молодец, написано от души

NightDawnFox committed:
Первый багфикс

NightDawnFox committed:
Обновление пожарного граба

NightDawnFox committed:
Быстрофикс проблемы

NightDawnFox committed:
Фиксы зелья

NightDawnFox committed:
упс

Rafnik02 commented:
Пришло время строчить клоун машин
